go/src/vendors/sonic/client/dbclient/counter_repository.go

```diff
@@ -0,0 +1,75 @@
+// Defines APIs to get interface counters from redis db - CountersDB.
+package dbclient
+
+import (
+	"strconv"
+	. "go/src/lib/lomcommon"
+)
+
+const (
```

Comment:
Generally can we prefer to have caps names for constants to differentiate from global variables ?

Reply:
Making them caps would expose them to other packages as well. Since they are not required from outside, hence kept it in small.